app/models/procedure_revision_preloader.rb

```diff
@@ -0,0 +1,56 @@
+class ProcedureRevisionPreloader
+  def initialize(revisions)
+    @revisions = revisions
+  end
+
+  def all
+    revisions = @revisions.to_a
+    load_revisions(revisions)
+  end
+
+  def self.load_one(revision)
+    ProcedureRevisionPreloader.new([revision]).all.first
+  end
+
+  private
+
+  def load_revisions(revisions)
+    revisions.map { load_procedure_revision_types_de_champ(_1) }
+  end
+
+  def load_procedure_revision_types_de_champ(revision)
+    prtdcs = ProcedureRevisionTypeDeChamp
```

Comment:
```suggestion
    coordinates = ProcedureRevisionTypeDeChamp
```

Je crois qu'on utilise plus dans le code `coordinate` que `prtdcs` pour désigner cet objet. Je suis d'accord que c'est discutable car c'est moins les initiales mais j'ai l'impression que ca explique assez simplement ce que signifie cet objet.

Reply:
merci je trouvais ça moche aussi. majé sur https://github.com/demarches-simplifiees/demarches-simplifiees.fr/pull/9922/commits/b178ebfabc05b9c4d185facce3ddaa69b396dc7a